src/Xamarin.Android.Build.Tasks/Tasks/AndroidUpdateResDir.cs

```diff
@@ -61,9 +62,10 @@ public class AndroidComputeResPaths : AndroidTask
 
 		public override bool RunTask ()
 		{
-			var intermediateFiles = new List<ITaskItem> ();
-			var resolvedFiles = new List<ITaskItem> ();
-			
+			bool caseSensitive = LowercaseFilenames || (Directory.Exists (IntermediateDir) ? Files.DirectoryIsCaseSensitive (IntermediateDir) : true);
```

Comment:
I'm inclined to remove `LowercaseFilenames ||` from this line for simplicity.

I originally included it because technically if `LowercaseFilenames` is `true`, then the destination files will all be lowercase, so it doesn't matter if the dictionary uses `StringComparer.Ordinal` or `StringComparer.OrdinalIgnoreCase`, and I believe `StringComparer.Ordinal` has a performance advantage.

But it looks like the performance of `StringComparer.OrdinalIgnoreCase` is fairly comparable in this case, so I think it might be OK to remove the `LowercaseFilenames ||` expression.

Example timings for 10,000 resource names with `StringComparer.Ordinal`:

    385 ms, 376 ms, 379 ms. Average: 380 ms

Example timings for 10,000 resource names `StringComparer.OrdinalIgnoreCase`:

    387 ms, 392 ms, 400 ms. Average: 393 ms